aiosmtpd/docs/NEWS.rst

```diff
@@ -42,7 +44,7 @@ Removed
 1.1 (2017-07-06)
 ================
 * Drop support for Python 3.4.
-* As per RFC 5321, §4.1.4, multiple ``HELO`` / ``EHLO`` commands in the same
+* As per RFC 5321, ยง4.1.4, multiple ``HELO`` / ``EHLO`` commands in the same
```

Comment:
Why is this changing?

Reply:
This is a coding error caused by the editor I'm using.
I'll fix it later.